### DISCUSSION TITLE
Send parameters from a controller to a view and how to use them?

### DISCUSSION THREAD

**Discussion by thibaultferaux** [statamic/cms#7626] [Q&A] [answered]
I want to send parameters from a controller to a antlers view, but when I try this following the documentation on controllers I can't access the parameters in my view.

Controller:
```
public function bookingStepTwo(Request $request, $id)
{
    return (new \Statamic\View\View)
        ->template('pages/booking2')
        ->layout('layout')
        ->with(['identifier', $id]);
}
```

View:
`<h2 class="text-center mb-8">{{ identifier }}</h2>`

The $id is not empty because when I dump it from the controller I get the correct value

**Comment by thibaultferaux** [comment, marked as answer]
I have found the answer.
In the array of the `with` function i have to use `=>` instead of `,`.
This was my fix.

**Reply by jasonvarga** [reply]
Yup. The array keys are the variable names you'll get in the template.

```php
->with([
  'identifier' => $id,
  'foo' => 'bar',
]);
```

```antlers
{{ identifier }} {{ foo }}
```
